src/services/addNote.ts

```diff
@@ -5,7 +5,7 @@ import CourtCase from "./entities/CourtCase"
 import insertNotes from "./insertNotes"
 
 const MaxNoteLength = 2000
-const notesRegex = new RegExp(`(.{1,${MaxNoteLength}})`, "g")
+const notesRegex = new RegExp(`(.|\\r|\\n){1,${MaxNoteLength}}`, "g")
```

Comment:
You can use `\s` instead which includes `.`, `\r`, and `\n`.

```suggestion
const notesRegex = new RegExp(`(\\s){1,${MaxNoteLength}}`, "g")
```